tools/unpacker/upx/features/certificate-info/test.py

```diff
@@ -16,7 +16,7 @@ def test_certificate_directory(self):
         first_sig = self.fileinfo.output['digitalSignatures']['signatures'][0]
         assert first_sig['digestAlgorithm'] == 'sha1'
         # Unpacked file -> different contents than original -> different hash -> invalid signature on the unpacked file
-        assert first_sig['fileDigest'] == '5CBE1AD2114B8EA09819F798DA2CBD89CAC4E53B'
+        assert first_sig['fileDigest'] == '3E8002A08AEB8A1AF564E26C84FD0352C1302FEA'
```

Comment:
Why did changes in import handling in https://github.com/avast/retdec/pull/1003 also changed this value related to certificates?

Reply:
This is not related to file certificates, but to the unpacked file data. There was a bug, introduced by me in the past when working on the image loader (ImportDirectory.h):
Old code:
```
td.itd.Ordinal = wHint /* | PELIB_IMAGE_ORDINAL_FLAGS::PELIB_IMAGE_ORDINAL_FLAG */;
```
New code:
```
td.itd.Ordinal = wHint | m_ordinalMask;
```
The code did not set the highest bit in order to indicate that it's an ordinal. As result, loader thought that it's a function name RVA and ended up in an error when mapping the PE.